src/index.d.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference types="node" />
+import { BuildOptions } from 'esbuild';
+
+export interface BundleOnceParams {
+  filePath: string;
+  outputPath: string;
+  esBuildUserOptions: BuildOptions;
+}
+export interface CypressFileObject {
+  filePath: string;
+  outputPath: string;
+  shouldWatch: boolean;
+}
+export declare type CypressPlugin = (file: CypressFileObject & NodeJS.EventEmitter) => Promise<string> | string;
+
+export interface BundleOnceParams {
```

Comment:
This is declared twice

Reply:
Thanks! I've commited now a fix on this.